src/search/evaluators/weighted_evaluator.cc

```diff
@@ -12,14 +12,11 @@ using namespace std;
 
 namespace weighted_evaluator {
 WeightedEvaluator::WeightedEvaluator(const Options &opts)
-    : evaluator(opts.get<shared_ptr<Evaluator>>("eval")),
+    : Evaluator(opts),
+      evaluator(opts.get<shared_ptr<Evaluator>>("eval")),
       w(opts.get<int>("weight")) {
 }
 
-WeightedEvaluator::WeightedEvaluator(const shared_ptr<Evaluator> &eval, int weight)
```

Comment:
It has. We now need to construct evaluators with an options object because the base class needs it. Thus having a second constructor without the options object is no longer possible.